sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-SBSD.e2e-spec.ts

```diff
@@ -0,0 +1,77 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executeAssessment,
+} from "../../test-utils";
+import { InstitutionTypes } from "../../models";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-SBSD.`, () => {
+  // Expected and not expected institution types.
+  const EXPECTED_INSTITUTION_TYPES = [
+    InstitutionTypes.BCPublic,
+    InstitutionTypes.BCPrivate,
+  ];
+
+  describe("Should determine SBSD as eligible when total assessment need is greater than or equal to 1, application PD/PPD status is 'yes' and", () => {
+    for (const institutionType of EXPECTED_INSTITUTION_TYPES) {
+      it(`institutionType is ${institutionType}`, async () => {
+        // Arrange
+        const assessmentConsolidatedData =
+          createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+        assessmentConsolidatedData.institutionType = institutionType;
+        assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+
+        // Act
+        const calculatedAssessment = await executeAssessment(
+          `parttime-assessment-${PROGRAM_YEAR}`,
+          assessmentConsolidatedData,
+        );
+
+        // Assert
+        expect(calculatedAssessment.variables.awardEligibilitySBSD).toBe(true);
+        expect(
+          calculatedAssessment.variables.finalProvincialAwardNetSBSDAmount,
+        ).toBeGreaterThan(0);
+      });
+    }
+  });
+
+  it(`Should determine SBSD as not eligible when institution type is ${InstitutionTypes.International}.`, async () => {
```

Comment:
The comment could be explicit about institution type. (BCPublic or BCPrivate)

Reply:
is it not understandable? 
![image](https://github.com/bcgov/SIMS/assets/77353155/185aeffc-9e62-4309-904b-990700c2a466)

Reply:
I later edited my comment mentioning we can run a test for bcpublic and bcprivate and one outside this for negative scenario.

Reply:
it's already happening, The first Tc of this test suite
![image](https://github.com/bcgov/SIMS/assets/77353155/7eabe6f8-25b4-4d57-a438-f3884bc354d3)

Reply:
I did not refresh the source code downloaded to local. All good now 😊